stylesheet.css

```diff
@@ -125,9 +125,26 @@ p>a {
     color: #000000;
 }
 
-#div_name2 {
-    display: none;
-    margin: 15px 15px 0px 15px;
-    padding: 5px;
+.grooming {
+    margin-bottom: 6em;
+}
+
+.container {
     border: 1px solid #aaa;
+    margin: 15px 15px 0 15px;
+    padding: 5px;
+}
+
+.hide {
+    display: none;
+}
+
+.show {
+    display: block;
+}
```

Comment:
The script we're using relies on setting/unsetting a show or hide class. It'll happily set those class names, but in order for setting those class names to do anything, they need to be defined in CSS.